packages/expo-cli/src/commands/build/ios/utils/image.ts

```diff
@@ -39,7 +38,8 @@ export async function ensurePNGIsNotTransparent(imagePathOrURL: string): Promise
         }
         try {
           // @ts-ignore: 'this' implicitly has type 'any' because it does not have a type annotation.
-          validateAlphaChannelIsEmpty(this.data, pick(this, ['width', 'height']));
+          const { data, width, height } = this;
+          validateAlphaChannelIsEmpty(data, { width, height });
```

Comment:
i personally wouldn't bother destructuring here, i'd just do: `validateAlphaChannelIsEmpty(data: this.data, { width: this.width, height: this.height });